plot_data/core.py

```diff
@@ -1419,9 +1402,9 @@ def bounding_box(plot_datas: List[PlotDataObject]):
     xmin, xmax, ymin, ymax = math.inf, -math.inf, math.inf, -math.inf
     for plot in plot_datas:
         if hasattr(plot, 'bounding_box'):
-            bb = plot.bounding_box()
-            xmin, xmax = min(xmin, bb[0]), max(xmax, bb[1])
-            ymin, ymax = min(ymin, bb[2]), max(ymax, bb[3])
+            bounding_box_ = plot.bounding_box()
```

Comment:
Is it necessary to escape `bounding_box` by `bounding_box_` here ?

Reply:
Yes because the function (and not method) name is bounding_box